### ISSUE TITLE
Dark mode support?

### ISSUE THREAD

**Issue by Greatz08** [ngoduykhanh/wireguard-ui#580]
Previous issue regarding this one got auto close by gitbot

**Comment by loyukfai** [comment]
Yes would be nice. When everything is in dark mode just one or two apps ain't, is problematic.

**Comment by samrocketman** [comment]
I have [opened a PR](https://github.com/ngoduykhanh/wireguard-ui/pull/671) which provides dark mode.  See PR for screenshots.

Side note: I am using it in my home assistant addon.  Since this addon doesn't appear to release often, the addon downloads source code and applies patches before building.
